src/simularium/rendering/CompositePass.ts

```diff
@@ -136,18 +136,19 @@ class CompositePass {
                 // contains IDs.  index into data buffer.
                 // typeId, instanceId, viewZ
                 vec4 col0 = texture(colorTex, texCoords);
+                // check for unitialized (set to clear value which was negative to indicate nothing drawn here to colorize)
                 if (col0.w < 0.0) {
                     discard;
                 }
                 float occ1 = texture(ssaoTex1, texCoords).r;
                 float occ2 = texture(ssaoTex2, texCoords).r;
-                int instanceId = int(col0.y);
+                float instanceId = (col0.y);
             
-                if(instanceId < 0)
+                if(instanceId < 0.0)
```

Comment:
for my own knowledge: is this check actually different? Or does typescript not like to compare a float to an integer. I would just think that `0.1 < 0` would be the same as `0.1 <0.0`

Reply:
Because there's no syntax highlighting here, it's harder to tell that this is GLSL code and not Typescript.  The variable type of instanceId used to be int and now it's float, and the comparison against integer 0 is a GLSL compile error.